src/v2/view-builder/views/TerminalView.js

```diff
@@ -2,10 +2,32 @@ import { createCallout, loc } from 'okta';
 import BaseView from '../internals/BaseView';
 import BaseForm from '../internals/BaseForm';
 import BaseFooter from '../internals/BaseFooter';
-import {getBackToSignInLink} from '../utils/LinksUtil';
+import BaseHeader from '../internals/BaseHeader';
+import HeaderBeacon from '../components/HeaderBeacon';
+import { getBackToSignInLink } from '../utils/LinksUtil';
+import { getIconClassNameForBeacon } from '../utils/AuthenticatorUtil';
 
 const RETURN_LINK_EXPIRED_KEY = 'idx.return.link.expired';
 
+const EMAIL_AUTHENTICATOR_TERMINAL_KEYS = [
+  'idx.transferred.to.new.tab',
+  'idx.return.to.original.tab',
+  RETURN_LINK_EXPIRED_KEY,
+  'idx.return.stale',
+  'idx.return.error',
+  'idx.email.verification.required'
+];
+
+const EMAIL_AUTHENTICATOR_TYPE = 'email';
+
+const HeaderBeaconTerminal = HeaderBeacon.extend({
```

Comment:
Can you attach some screenshots of what this looks like with and without email terminal response?

Reply:
I've attached before and after screenshots..